smriprep/cli/run.py

```diff
@@ -55,6 +55,8 @@ def get_parser():
     g_bids.add_argument('--participant-label', '--participant_label', action='store', nargs='+',
                         help='a space delimited list of participant identifiers or a single '
                              'identifier (the sub- prefix can be removed)')
+    g_bids.add_argument('--bids_filters', action='store', type=Path,
```

Comment:
We use hyphens instead of underscores. This has no effect on the variable names, though.

```suggestion
    g_bids.add_argument('--bids-filters', action='store', type=Path,
```